### DISCUSSION TITLE
Stable version for .NET 5

### DISCUSSION THREAD

**Discussion by shirinp** [UiPath/corewf#183] [Q&A] [answered]
Hi,
Do we have any stable version which is compatible with .NET 5 or lower .NET CORE version ?

**Comment by lbargaoanu** [comment, marked as answer]
Yes, just choose the latest version that targets .Net 5.

**Reply by fubar-coder** [reply]
I guess what he mans is, that it'd be nice to have a CoreWF version without the "-20211103-03" suffix, because that automatically makes it a prerelease version (according to SemVer 2.0) and some projects/project managers may forbid using NuGet packages with prerelease versions when bundling a release (package).
